CMake/HPHPFindLibs.cmake

```diff
@@ -510,7 +510,11 @@ macro(hphp_link target)
   if (LIBZIP_LIBRARY)
     target_link_libraries(${target} ${LIBZIP_LIBRARY})
   else()
-    target_link_libraries(${target} zip_static)
+    add_dependencies(${target} zip_static)
+    target_link_libraries(${target} "${CMAKE_CURRENT_SOURCE_DIR}/third-party/libzip/src/lib/libzip.a")
```

Comment:
This doesn't work (I'm probably using `${CMAKE_CURRENT_SOURCE_DIR}` incorrectly here) but I wanted to see if you had any better suggestions for how to correctly link the library before I spend too much time fixing the exact paths...